### PR TITLE
Add maplibre-rs news

### PR DESCRIPTION
**Pull request by maxammann**

### CODE REVIEW

File: content/news/2023-06-07-maplibre-rs-monthly.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+The governing board by vote! If you did not yet know, MapLibre is governed by an [elected](http://localhost:1313/about/).
+See [here](https://github.com/maplibre/maplibre/pull/282) to have more insight on the process.
+
+---
+
+Back to the progress in maplibre-rs.
+A lot has happened in the past half year. I have been focusing on getting the plugin architecture in a working state. It is now possible to enable or disable features like raster or vector rendering at compile time and runtime. This means that features are now well separated.
+
+Right now, I am focusing on getting the projection correct in [this PR](https://github.com/maplibre/maplibre-rs/pull/233/). Unfortunately, there is quite some match involved and the maplibre-gl-js source code does not explain much.
```
Comment (birkskyum) [inline content/news/2023-06-07-maplibre-rs-monthly.md:42]: match -> math

File: content/news/2023-06-07-maplibre-rs-monthly.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+So in order to get this correct (and yes we sadly have to get this 100% correct) I will have to do some math on paper sadly and then create unit tests to validate my assumptions.
```
Comment (birkskyum) [inline content/news/2023-06-07-maplibre-rs-monthly.md:43]: 2 x sadly.. sounds a bit depressing.

**Review by birkskyum — COMMENTED** [review]
In general a good article - it's great to get a post about the project again. There is quite a lot of "I", which indicate it's a one-person effort, and if it's intended to become more of a collaborative effort I'd suggest to be aware of that type of communication to make sure the project is inviting.

**Comment by maxammann** [comment]
> In general a good article - it's great to get a post about the project again. There is quite a lot of "I", which indicate it's a one-person effort, and if it's intended to become more of a collaborative effort I'd suggest to be aware of that type of communication to make sure the project is inviting.

I replaced one I with we. I generally agree, though in this blog post I actually go over the things where I've been focusing on. So there is no way I can replace it with we :/

**Comment by maxammann** [comment]
rdy for merge

**Comment by maxammann** [comment]
Can somebody merge? :)

**Comment by birkskyum** [comment]
@duje, would you prefer to hold this one too till the twitter creds are resolved?

**Comment by maxammann** [comment]
@duje ping

**Comment by maxammann** [comment]
> We're struggling with some twitter permissions - To avoid letting it block the pipeline completely, I think we should let this article out, and await twitter for the JAWG post.

We can always post it in one week on twitter :)

**Comment by wipfli** [comment]
Thanks for the nice blog post, @maxammann. I would also recommend to use 3rd person voice in the future. Sounds more appropriate for a big collaboration like MapLibre...

**Comment by maxammann** [comment]
Idk, I think this probably depends on the content. No need for a general rule.

It feels wrong to write in the 3rd person if there were no contributions from outside which I'm reporting about.

This was was a lot about a specific PR I'm working on.

**Comment by birkskyum** [comment]
> It feels wrong to write in the 3rd person if there were no contributions from outside which I'm reporting about.

It's only fair that you get a lot of credit for all this amazing work you put in, and I don't see a line of people who are willing and capable to do this work if you didn't, so it's in no way because it's not appreciated.

What I think Oliver and I, and arguably a lot of readers, might pick up on is this vantage point that you are the `inside` and everyone else is the `outside`, but it's probably just a symptom of the early one-person-effort state the project is in.

**Comment by maxammann** [comment]
Got it, yeah I can definitely change the tone.

So my previous somment was not about taking credit, but just that I don't like talking about myself in 3rd person :D

So that is why that blog post is a mix between two voiced, separated by paragraphs.

Even if I kept 1st person, the amount would differ per post.

I might move part of the blog post to my personal blog and then continue having half-yearly updates on the maplibre blog.

**Comment by maxammann** [comment]
I think an important part here, maybe the most important: is this a blog or a news section?


If it is news I agree it should be 3rd person all the way.

If it is a blog 1st person is actually preferred for most blogs.

**Comment by birkskyum** [comment]
The main role of `news` is to accommodate our newsletter, with a scope spanning the entire organization every month - I think we need a separate blog where there could be in-depth stories, and even thorough posts on new releases, which could then be linked from the newsletter.

**Comment by maxammann** [comment]
I agree. There is also the inside Rust blog which  is a more classical blog: https://blog.rust-lang.org/inside-rust/index.html

Imo a blog should not consist of announcements. In that case it is a newsletter or just a news page.

A blog usually contains a technical thing accompanied by a story. Reading it is fun and not just for the sake of news. Sometimes also a little bit of boasting about how cool a project is :D

**Comment by wipfli** [comment]
I see it more as a news section than a blog section.

**Pull request merged**